src/js/sdk.tsx

```diff
@@ -126,6 +126,7 @@ export function init(passedOptions: ReactNativeOptions): void {
                 ? `${appPrefix}${frame.filename}`
                 : `${appPrefix}/${frame.filename}`;
           }
+          delete frame.abs_path;
```

Comment:
M: `abs_path` is part of the protocol, so symbolication should work anyway, for fixing this issue is fine as of now, but I wonder if we should not fix ingestion when both `filename` and `abs_path` is provided? can you double check with the stack trace team?

Reply:
[Relay does exactly that](https://github.com/getsentry/relay/blob/1e470ec87aee8fb9214844a1cf5eaa5257ea3d69/relay-general/src/store/normalize/stacktrace.rs#L27-L29) it copies it from filename, so until we need the two fields in the SDK I think we can save a bit the envelope size and send only the filename.

Reply:
The reason why it didn't work is becuae the default frame parse now copies the name before the changes needed for RN. 

I've talked to the web team and I still don't exactly understand what they use it for, but we can adjust that later.

Reply:
So it's used for identifying new Debug Meta Image Type Source Maps, I will have to talk more to the web team on how it works.

https://develop.sentry.dev/sdk/event-payloads/debugmeta/#source-map-images

Reply:
alright, let's create a repo maintenance issue for that, so we don't forget it.

Reply:
https://github.com/getsentry/sentry-react-native/issues/2892